docs/src/environments/_index.md

```diff
@@ -54,106 +89,91 @@ Each new environment you create is considered a **child** of the **parent** envi
 
 Each child environment can [sync](../other/glossary.md#sync) code and/or data down from its parent
 and [merge](../other/glossary.md#merge) code up to its parent.
-The child environments are used for development, staging, and testing.
+You can use child environments for development, staging, and testing.
 
 When you [branch](../other/glossary.md#branch) an environment to create a new child environment,
 its parent is the environment it was created from.
 If you push a branch through Git or a [source integration](../integrations/source/_index.md),
-the parent defaults to the default environment.
+the parent environment is your [default environment](#default-environment).
 
 You can always [change an environment's parent](./change-parent.md).
 
-## Workflows
+### Workflows
+
+Since you can customize your [environment hierarchy](#hierarchy), you can create your own workflows. 
+You don't have to follow any rules when branching environments.
+You can decide which structure best fits your needs. 
 
-Since you can organize your environments as you want, you can create your own workflows.
-There are no rules you must follow when branching environments.
+#### Possible approaches
 
-You can choose a structure that might best fit your workflow.
-Some possible approaches:
+You may want to take one of the following approaches when creating your workflows:
 
-- **Agile**: A child environment per sprint.
-  Each story in the sprint can have its own environment as a child of the sprint environment.
-- **Developer-centric**: One QA environment and a few development environments
-  (*per developer*, *per task*, or similar).
-- **Testing**: An operational test environment, a user test environment, and a few unit test environments.
-- **Hotfix**: One environment for every bug, security, or hotfix that needs deployment.
+-   **Agile**: A child environment per sprint.
+    Each story in the sprint can have its own environment as a child of the sprint environment.
 
-Here is an example of a possible Agile workflow.
+-   **Developer-centric**: One QA environment and a few development environments
+    (*per developer*, *per task*, or similar).
 
-The administrator started by [branching](../other/glossary.md#branch) the Live environment to create a Sprint environment.
-They then give each developer permission to branch the Sprint environment to create new feature environments.
+-   **Testing**: An operational test environment, a user test environment, and a few unit test environments.
 
-![Agile branches with Live as the top parent, Sprint as a child, and Feature 1 and Feature 2 as children of Sprint](/images/workflow/branches.svg "0.2")
+-   **Hotfix**: One environment for every bug, security, or hotfix that needs deployment.
```

Comment:
:hammer_and_wrench: This is trying to get a parallel with `fix` with each part in the list, but it doesn't really work when `hotfix` is a single word.
```suggestion
-   **Hotfix**: One environment for each bug, security issue, or hotfix that needs deployment.
```